dev/breeze/src/airflow_breeze/utils/docs_publisher.py

```diff
@@ -66,6 +66,8 @@ def _current_version(self):
         if self.package_name.startswith("apache-airflow-providers-"):
             provider = get_provider_distributions_metadata().get(get_short_package_name(self.package_name))
             return provider["versions"][0]
+        if self.package_name == "task-sdk":
```

Comment:
Should this be "apache-airflow-task-sdk"? Don't think it really matters mind you, but that is what the dist is called

Reply:
Not too related. Those are not distribution names, they are names of "documentation packages" - which are different than distributions - they also include "docker-stack" , "helm-chart", "apache-airflow-providers" and "apache-airflow" - and essentially they are the prefixes in our documentation pages:


* https://airflow.apache.org/docs/apache-airflow/stable/index.html
* https://airflow.apache.org/docs/apache-airflow-providers/index.html
* https://airflow.apache.org/docs/docker-stack/index.html
* https://airflow.apache.org/docs/helm-chart/stable/index.html


So they do not **have to" be distribution names and I'd say shorter in this case is better.